test_util/aws.py

```diff
@@ -0,0 +1,113 @@
+#!/usr/bin/env python3
+import logging
+from collections import namedtuple
+
+import boto3
+
+LOGGING_FORMAT = '[%(asctime)s|%(name)s|%(levelname)s]: %(message)s'
+logging.basicConfig(format=LOGGING_FORMAT, level=logging.DEBUG)
+logger = logging.getLogger(__name__)
+
+Host = namedtuple('Host', ['private_ip', 'public_ip'])
+
+
+class BotoWrapper():
+    def __init__(self, region, aws_access_key_id, aws_secret_access_key):
+        self.region = region
+        self.session = boto3.session.Session(
+            aws_access_key_id=aws_access_key_id, aws_secret_access_key=aws_secret_access_key)
+
+    def client(self, name):
+        return self.session.client(service_name=name, region_name=self.region)
+
+    def resource(self, name):
+        return self.session.resource(service_name=name, region_name=self.region)
+
+    def create_key_pair(self, key_name):
+        """Retruns private key of newly generated pair
+        """
+        key = self.resource('ec2').KeyPair(key_name)
+        return key.key_material
+
+    def delete_key_pair(self, key_name):
+        self.resource('ec2').KeyPair(key_name).delete()
+
+    def create_stack(self, name, template_url, user_parameters, deploy_timeout=60):
+        """Returns boto stack object
+        """
+        logging.info('Requesting AWS CloudFormation...')
+        cf_parameters = []
+        for k, v in user_parameters.items():
+            cf_parameters.append({'ParameterKey': k, 'ParameterValue': v})
+        self.resource('cloudformation').create_stack(
+            StackName=name,
+            TemplateURL=template_url,
+            DisableRollback=True,
+            TimeoutInMinutes=deploy_timeout,
+            Capabilities=['CAPABILITY_IAM'],
```

Comment:
nit: capabilities is technically template-specific (DCOS stack will want it, but the VPC stacks don't need it), but we can move that later when the code actually needs it